command/operator_debug.go

```diff
@@ -246,6 +246,13 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	}
 	c.interval = i
 
+	// Sanity check interval
+	if i.Seconds() > d.Seconds() {
+		c.Ui.Info(fmt.Sprintf("Interval %s is greater than duration %s, resetting to %s.", interval, duration, duration))
```

Comment:
I'm wondering if we really should autocorrect this rather than just returning an error: if someone has mistyped this, do we really know whether either the interval or the duration were the correct one? If we just return an error, they can see that and fix it. Whereas if we don't they can miss the warning and get results they didn't really want.

Reply:
Initially I considered it as a helper, but it's a fair point to force the operator to stop and make adjustments to achieve the intended outcome.  I fixed the associated test errors, and added an explicit test to cover the new validation.